js/zb.js

```diff
@@ -458,6 +459,11 @@ module.exports = class zb extends Exchange {
         return this.parseOrders (response, market, since, limit);
     }
 
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let orders = await this.fetchOrders (symbol, since, limit, params);
+        return this.filterBy (orders, 'status', 'closed');
```

Comment:
If the filtering is done on client side entirely, then we should not declare that their `fetchClosedOrders` endpoint is `true`, so, it could be `'emulated'`, however, this type of emulation breaks the pagination, so, we probably, should not add a non-existing method just for the filtering. If the user needs to – he could filterBy in userland, so, we don't wrap a new method just for that in any other exchange. Instead, it's better to tell the truth, which is: their `fetchClosedOrders` is really `false`. This way we would keep it much more controllable for the user in the first place. Do you agree?

Reply:
ok, thank you, I understand

Reply:
@ndubel thx to you for your help! Looking forward to seeing more ) Best wishes!